Makefile.in

```diff
@@ -0,0 +1,46 @@
+BINDIR = @BINDIR@
+LIBDIR = @LIBDIR@
+SYSCONFDIR = @SYSCONFDIR@
+MANDIR = @MANDIR@
+DATADIR = @DATADIR@
+NAME = @NAME@
+VERSION = @VERSION@
+
+.DEFAULT_GOAL := build
+.PHONY: build
+build: debbuild.out
+
+debbuild.out: debbuild debbuild.spec
+	# Transfer $VERSION into the live system
+	perl -pe "s/\@VERSION\@/$(VERSION)/" debbuild > debbuild.out
+	chmod +x debbuild.out
+
+.PHONY: check
+check:
+	yath
+
+.PHONY: install
+install:
+	rm -rf $(DESTDIR)
+
+	install -d $(DESTDIR)$(BINDIR)
+	install debbuild.out $(DESTDIR)$(BINDIR)/debbuild
+
+	install -d $(DESTDIR)$(LIBDIR)/$(NAME)
+	install -m 644 macros/macros.in $(DESTDIR)$(LIBDIR)/$(NAME)/macros
+	install -m 644 config/debrc $(DESTDIR)$(LIBDIR)/$(NAME)
+
+	install -d $(DESTDIR)$(SYSCONFDIR)/$(NAME)
+	install -m 644 macros/macros.sysutils $(DESTDIR)$(SYSCONFDIR)/$(NAME)
+	install -m 644 macros/macros.texlive $(DESTDIR)$(SYSCONFDIR)/$(NAME)
+	install -m 644 macros/platform.in $(DESTDIR)$(SYSCONFDIR)/$(NAME)/macros
```

Comment:
Yeah, and that was generally wrong to begin with.

Reply:
I've been trying not to change runtime functionality with this PR, and changing where a config file is installed definitely feels like it would do so. Would you be OK with my making that change in a different PR instead?

Reply:
Yeah, that's fine.